### PR TITLE
Resize tensor

### PR DESCRIPTION
**Pull request by truongsinh**
I did consider a different API, `Tensor.resize(shape)`, but it's complicated to let `tensor` knows `interpreter`, and inconsistent with Python API.

### CODE REVIEW

**Comment by truongsinh** [comment]
test added

File: test/tflite_test.dart [hunk 1]
```diff
@@ -113,6 +119,11 @@ void main() {
       expect(interpreter.getOutputTensors(), hasLength(1));
     });
 
+    test('resize unput tensor', () {
```
Comment (lambdabaa) [inline test/tflite_test.dart]: nit: input*

**Comment by lambdabaa** [comment]
@truongsinh After CI is passing this LGTM, thanks!

**Pull request merged**